src/app/app.filter.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+    name: 'filterSearch'
+})
+
+export class FilterSearch implements PipeTransform {
+    transform(item: string, search: string) {
+        if (item && search) {
+            const contains = new RegExp(search, 'gi');
+            if (contains.test(item)) {
+                return true;
```

Comment:
Minor note: can we simplify this chunk with "return contains.test(item)" instead if-statement?